### PR TITLE
Migrate xsd file URI scheme from http to https

### PR DESCRIPTION
**Pull request by rbrud**
The change eliminates the 301 Moved Permanently redirect

Additional change includes updating the TopoGrafix version from 1.0 to 1.1 in the read and write example

### CODE REVIEW

**Comment by twpayne** [comment]
Thanks for opening this! It looks like a couple more tests need updating. The lint failure is unrelated to this PR. I'll update the project infrastructure and then ask you to rebase this PR.

**Comment by twpayne** [comment]
Following #36, please can you rebase this PR on the latest `master`.

**Comment by rbrud** [comment]
Thanks for considering this pull request. I have rebased the fork.

File: gpx.go [hunk 1]
```diff
@@ -16,6 +16,8 @@ import (
 )
 
 const timeLayout = time.RFC3339Nano
+const http = "http://"
+const https = "https://"
```
Comment (twpayne) [inline gpx.go:20]: These three lines need to be written as

```go
const (
    timeLayout = time.RFC3339Nano
    http = "http://"
    https = "https://"
)
```

to be idiomatically formatted (lint check).

**Review by twpayne — COMMENTED** [review]
There are still a couple of examples that need fixing.

Run

    go test ./...

to see the failing tests locally.

**Comment by rbrud** [comment]
Eureka! I squashed my commits locally.

```
$ go test ./...
?       github.com/twpayne/go-gpx/cmd/gpxdump   [no test files]
ok      github.com/twpayne/go-gpx       0.037s
```

```
$ git diff master --name-only
README.md
gpx.go
gpx_examples_test.go
gpx_test.go
testdata/ashland.gpx
testdata/fells_loop.gpx
testdata/mystic_basin_trail.gpx
```

**Comment by twpayne** [comment]
Thanks! PR merged.

I've tagged `v1.4.0` that includes your changes.

**Comment by rbrud** [comment]
Thanks for accepting my pull request! I really appreciate it.

**Pull request merged**